README.md

```diff
@@ -1 +1,75 @@
-# Budget_App
\ No newline at end of file
+# Budget App
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 Budget APp ](#-blog-app-)
+  - [🛠 Built With ](#-built-with-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Run tests](#run-tests)
+  - [👥 Authors ](#-authors-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 BUDGET App <a name="about-project"></a>
+
+> "Budget App" is a mobile web application that gives users the power to manage their budget: users can have a list of expenses (e.g bed) associated with a broader category(Home) so that they can know how much they spent on what.
```

Comment:
Good job on your Ruby on Rails capstone project, good job so far 👍

*Feedback about your video presentation.*

- [ ] In your [video presentation](https://www.loom.com/share/c6f94bc747a543cd8deeaaffd274b475), you didn't highlight the piece of the code that you find interesting or proud of making. 🤓

According to the project requirements, The content of the video should include:

1. The student recorded a video presenting the features of the project. ✅ 
2. The maximum length of the video is 5 minutes. ✅ 
3. The video contains a clear and objective description of the project in professional English. The topic and purpose of the project are made clear. ✅ 
4. The video contains a demo of the project, highlighting all the project features. ✅ 
5. The video contains a highlight about a piece of the code the student finds interesting or is proud of making. ❌ 


I would recommend that you record a new video that includes all 5 details as requested in the project requirement 🛠👍